sdk/errors.go

```diff
@@ -0,0 +1,35 @@
+package sdk
+
+import (
+	"errors"
+
+	mongodbatlasv2 "go.mongodb.org/atlas/mongodbatlasv2"
+)
+
+// GetAPIError gets a strongly typed error from an error.
+func GetAPIError(err error) *mongodbatlasv2.Error {
```

Comment:
```suggestion
// APIError gets a strongly typed error from an error.
func APIError(err error) *mongodbatlasv2.Error {
```
[no getters](https://go.dev/doc/effective_go#Getters)